### PR TITLE
Fixes #289: Offer copying the text in the MailtoDropdown manually 

### LINKED ISSUE #289: MailtoDropdown: Offer copying the text manually
In addition to the regular `mailto` link and the webmailers, it should be possible to get a modal displaying the subject, recipient and body that the user can copy manually.

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

**Comment by baltpeter** [comment]
Fixed some problems with the tests. This can be squashed when merging.

File: src/Components/MailtoDropdown.js [hunk 1]
```diff
@@ -24,6 +31,77 @@ export const MAILTO_HANDLERS = {
...
+                            <input
+                                type="text"
+                                id="mailto-dropdown-copymanually-subject"
+                                className="form-element"
+                                value={decodeURIComponent(d.subject)}
+                                onClick={onInputClick}
+                                readOnly
+                            />
+
+                            {d.email
```
Comment (zner0L) [inline src/Components/MailtoDropdown.js:69]: I actually think conditionally hiding this is confusing. Why not just leave it empty? When I tested I was cunfused whether it was missing. An empty input would hint to users that they need to select a recipient.

File: src/Components/MailtoDropdown.js [hunk 2]
```diff
@@ -49,10 +127,13 @@ export default class MailtoDropdown extends preact.Component {
         };
         const handler_buttons = handlers.map(h => (
             <a
-                href={MAILTO_HANDLERS[h].link(data)}
+                href={MAILTO_HANDLERS[h].link && MAILTO_HANDLERS[h].link(data)}
```
Comment (zner0L) [inline src/Components/MailtoDropdown.js:130]:
```suggestion
                href={MAILTO_HANDLERS[h].link?(data)}
```
After rebasing that can be simplified.

File: src/Components/MailtoDropdown.js (hunk 1 above)
Comment (baltpeter) [reply src/Components/MailtoDropdown.js:69]: Hm, interesting. Personally I think showing the empty input looks quite stupid.

But I mean, eh. It's not the end of the world. Sure, why note.

File: src/Components/MailtoDropdown.js (hunk 2 above)
Comment (baltpeter) [reply src/Components/MailtoDropdown.js:130]: Actually, that's `href={MAILTO_HANDLERS[h].link.?(data)}` as per [1](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining.html#calling-deeply-nested-functions). :)

**Pull request merged**